clusters/app.ci/release-controller/deploy-ocp-priv-controller.yaml

```diff
@@ -31,8 +31,7 @@ spec:
 apiVersion: v1
 kind: Service
 metadata:
-  annotations:
-    service.alpha.openshift.io/serving-cert-secret-name: release-controller-priv-tls
+  annotations: {}
```

Comment:
You can just remove this.

Reply:
This yaml file, together with other release controller yamls, is controlled by the yaml.
Let us prove it is working with the oauth container.